tests/istioctl_analyze.sh

```diff
@@ -86,5 +86,11 @@ assert_contains_string error.txt "Referenced gateway not found"
 
 testcase "kpt_istioctl_analyze_declarative_example"
 kpt pkg get "$CATALOG_REPO"/examples/istioctl-analyze .
+kpt fn run istioctl-analyze 2>err.txt || true
+assert_contains_string err.txt "Schema validation error: gateway must have at least one server"
+
+# TODO: Remove after fixing https://github.com/GoogleContainerTools/kpt/issues/983
```

Comment:
The assertion is same for both tests (unlike kubeval). Revert changes to istioctl_analyze.sh, you don't want to be testing kpt CLI here, just the functions.

Reply:
Done